wremnants/theory_tools.py

```diff
@@ -375,16 +376,18 @@ def make_theory_corr_hists(df, name, axes, cols, helpers, generators, modify_cen
 
     return res
 
-def scale_angular_moments(hist_moments_scales, sumW2=False, createNew=False):
+def scale_angular_moments(hist_moments_scales, upToA4= False, sumW2=False, createNew=False):
```

Comment:
I don't think there's much point in restricting upToA4, just index the histogram after you make it

Reply:
Yes this should by removed